### PR TITLE
Add instructions for SSH (and making udev rules)

### PR DESCRIPTION
**Pull request by UniversalSuperBox**
Small edit, but it's been on my to-do list for too long.

### CODE REVIEW

File: porting/debug-build/index.rst [hunk 1]
```diff
@@ -62,6 +62,41 @@ Now you have terminal access to the system running from initrd.
 
 The ``hybris-boot.img`` brings up the telnet interface if it detects a problem. Alternatively, you can use the ``hybris-recovery.img`` image which will always start telnet.
 
+Debugging via SSH
```
[the remark this answers is not in the document]
Comment (UniversalSuperBox) [reply porting/debug-build/index.rst]: Sure, that could work.

File: porting/debug-build/index.rst [hunk 2]
```diff
@@ -62,6 +62,41 @@ Now you have terminal access to the system running from initrd.
...
+You may now reboot your device into Halium.
+
+Once your device is booted, you should see that you have a new network interface again. Assign it an IP of 10.15.19.100. Once finished, you should be able to do the following to log in::
+
+    ssh root@10.15.19.82
+
+Add udev rules
+--------------
+
+Now that you're logged in, you must create some udev rules to allow some tests to access your hardware. Run the following command, replacing [codename] with your device's codename.::
```
[the remark this answers is not in the document]
Comment (bhush9) [reply porting/debug-build/index.rst]: There are no specifics.. some devices require the udev rules for even normal test like test_vibrator and some don't require at all.

File: porting/debug-build/index.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (bhush9) [reply porting/debug-build/index.rst]: +1

**Comment by UniversalSuperBox** [comment]
Are these changes better?

**Pull request merged**